libs/protos/protos/sp_external.proto

```diff
@@ -96,6 +97,18 @@ service External {
 
   rpc AppRegisterReject(AppRegisterRejectRequest) returns (StandardResponse);
 
+  // BEGIN Wasm methods
+  rpc GetWasm(GetWasmRequest) returns (GetWasmResponse);
+
+  rpc GetAllWasm(GetAllWasmRequest) returns (GetAllWasmResponse);
+
+  rpc CreateWasm(CreateWasmRequest) returns (StandardResponse);
```

Comment:
Create methods should return a custom response so we can include the ID of the new resource